includes/utils/class-amp-image-dimension-extractor.php

```diff
@@ -172,38 +172,7 @@ private static function determine_which_images_to_fetch( &$dimensions, &$urls_to
 	 * @param string $mode Whether image dimensions should be extracted concurrently or synchronously.
 	 */
 	private static function fetch_images( $urls_to_fetch, &$images, $mode ) {
-		// Use FasterImage when for compatible PHP versions
-		if ( 'synchronous' === $mode ||
```

Comment:
What about the `$mode` parameter? Shouldn't it be eliminated?

Reply:
There is also a `$mode` argument on `\AMP_Image_Dimension_Extractor::extract_by_downloading_images()`. Should an error be raised if it is anything other than `concurrent`? Should it be removed?